ReverseTicTacToe/Logic/Game.cs

```diff
@@ -164,6 +165,13 @@ private void makeMove(int i_Row, int i_Col)
             }
         }
 
+        public char GetCurrentPlayersChar()
+        {
+            return (m_CurrentTurn == eTurns.Player1) ? m_Player1Char : m_Player2Char;
```

Comment:
I just know we will get a comment on that, maybe extand?

Reply:
not sure what you mean